CHANGELOG_PENDING.md

```diff
@@ -3,3 +3,4 @@
 ### SDK Enhancements
 
 ### SDK Bugs
+The logging behavior in `aws/ec2metadata/token_provider.go` was updated: warnings about falling back to IMDSv1 are now logged only when LogLevel is set to `LogDebugWithDeprecated`. This change prevents unnecessary warnings when LogLevel is set to suppress messages.
```

Comment:
There should be 2 space indent IIRC. go v1's changelog has such strict format limit

Reply:
e.g.

```
### SDK Bugs
* line 1
  * line 2
 ```